src/main/java/game/map/Map.java

```diff
@@ -77,4 +80,42 @@ public List<Player> getPlayers() {
         return players;
     }
 
-}
+        public static HashMap<Player, List<Country>> assignCountries(List<Player> players, List<Country> countries) {
```

Comment:
I don't think we have to return anything. This could be a `void` method.

Reply:
completed